nux/store/selectors.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { includes, difference, keys } from 'lodash';
+
+/**
+ * Returns an object describing the guide, if any, that the given tip is a part
+ * of.
+ *
+ * @param {Object} state Global application state.
+ * @param {string} tipID The tip to query.
+ *
+ * @typedef {Object} NUX.GuideInfo
+ * @property {string[]} tipIDs       Which tips the guide contains.
+ * @property {?string}  currentTipID The guide's currently showing tip.
+ * @property {?string}  nextTipID    The guide's next tip to show.
+ *
+ * @return {?NUX.GuideInfo} Information about the associated guide.
```

Comment:
Where can I read more about this `NUX.GuideInfo` type ? (We should want to [`@typedef`](http://usejsdoc.org/tags-typedef.html) any of our custom types)

Reply:
It's defined above. I've moved the `@typedef` into its own block in 6af12cbd2